src/lib.rs

```diff
@@ -290,11 +293,18 @@ mod tests {
     fn test_bad_signatures_fails() {
         let plaintext = Bytes(vec![69, 42, 0]);
 
-        let alice = mnemonic_to_pair(&new_mnemonic()).unwrap();
+        let alice_seed =
+            hex::decode("e5be9a5092b81bca64be81d212e7f2f9eba183bb7a90954f7b76361f6edb5c0a")
```

Comment:
I'd prefer if these were part of a Keyring enum or even consts somewhere, but not a huge deal if you don't want to fix that in this PR